### ISSUE TITLE
Support (or improve support for) attaching debugger to the childprocesses

### ISSUE THREAD

**Issue by thomcc** [google/evcxr#95]
I'm not sure what this would look like really, but in #93 it would have been nice to be able to attach lldb .

The ideal situation would be if `:debugger <expr>` to attach `rust-lldb`, set a breakpoint right before evaluating the expr, and drop me into the debugger prompt. Not sure how doable this is though.

Another (maybe easier) option would be for `:debugger <expr>` to spawn the child process, and print out the PID, and the function I should set a breakpoint on, but then wait for me to hit `<return>` (or something indicating i've attached the debugger) before actually evaluating `<expr>`.

**Comment by davidlattimore** [comment]
Good idea. I'll give it some thought. Sounds like it should be achievable though
